src/components/modals/authentification/ModalPasswordForget.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * Modal avec formulaire pour reinitialiser son compte
+ */
+
+import { sendEmailToResetPassword } from '@/api/authentification';
+import { SubmitHandler, useForm } from 'react-hook-form';
+
+type LoginFormTypeForget = {
+  email: 'string';
+};
+
+type Props = {
+  setContenuModal: React.Dispatch<
+    React.SetStateAction<
+      'init' | 'connexion' | 'inscription' | 'password-forget'
+    >
+  >;
+};
+
+const ModalPasswordForget = ({ setContenuModal }: Props) => {
+  const { handleSubmit, register, reset } = useForm<LoginFormTypeForget>();
+
+  const onSubmit: SubmitHandler<LoginFormTypeForget> = async (data) => {
+    handleResetPassword(data);
+  };
+
+  const handleResetPassword = async ({ email }: LoginFormTypeForget) => {
+    const { error } = await sendEmailToResetPassword(email);
+
+    if (error) {
+      console.error(error.message);
+      return;
+    }
+    reset();
+    setContenuModal('init');
+  };
+
+  return (
+    <>
```

Comment:
Message de Sonarlint : A fragment with only one child is redundant.sonarlint(typescript:S6749)

Reply:
corrigé